pkgs/by-name/fo/forgejo/generic.nix

```
@@ -40,9 +40,9 @@ let
     pname = "forgejo-frontend";
     inherit src version npmDepsHash;
 
-    patches = [
-      ./package-json-npm-build-frontend.patch
-    ];
+    buildPhase = ''
+      ./node_modules/.bin/webpack
+    '';
```

Nice touch getting rid of the patch instead of rebasing it! I did not consider that!

---

Usually we would have to add `runHook` as well, but I'd argue this is fine as is, given you can't easily reach the `frontend` derivation to override it anyway.
  
```suggestion
    buildPhase = ''
      runHook preBuild
    
      ./node_modules/.bin/webpack
      
      runHook postBuild
    '';
```

Not blocking, just a fyi. Keep it as is if you want :)